docs/reference/targets/adhoc_tool.mdx

```diff
@@ -1,5 +1,7 @@
 ---
 title: adhoc_tool
+# The old-style `code...code` anchors are heading level 6, and shouldn't be visible in ToC
```

Comment:
Did you see/ did you try just using h tags so they don't show up in ToC?

https://docusaurus.io/docs/markdown-features/toc#customizing-table-of-contents-generation

Reply:
Yep, it has the same positioning problem as the original version. You can see the minor variations of this current state described in the big comment in the field template: https://github.com/pantsbuild/pantsbuild.org/pull/55/commits/6bb8ab687de358fb8c7332d858e00b18c8a4c03c

(I note the comment refers to `<A />` but the same applies to any HTML-tag-based method.)